src/aims_ui/templates/custom_address_info/_macro.njk

```diff
@@ -0,0 +1,84 @@
+{% macro customAddressInfo(params) %}
+{% from 'components/label/_macro.njk' import onsLabel %}
+
+{% for address in params.matched_addresses %}
+
+<article class="result-short-form">
+<a href="url"><h2 class="saturn"> {{ address.formattedAddress.value }}</h2></a>
+<table> <tbody>
+
+{% for attribute_name, address_attribute in address.__dict__.items() %}
+{% if address_attribute.show %}
+	<tr><td class="mars">{{ address_attribute.name }}</td>
+	    <td class="venus">{{ address_attribute.value }}</td>
+	</tr>
+{% endif %}
+{% endfor %}
+<tr>
+
+    <td class="mars"><a href=" {{ 'https://www.google.com/maps/search/?api=1&query=' + address.geo.value.get('longitude') + ',' + address.geo.value.get('latitude')  }}"
+    >Location Map</a></td>
+
+</tr>
+
+</tbody> </table>
+</article>
+
+
+
+
+
+
+
+
+
+<style>
```

Comment:
Style information generally shouldn't be hardcoded - is this normal for templates?

Reply:
In this case, the original UI appears to have a completely different set of styles for this page, as shown below for each style of title. This seemed to be the best way to reproduce that for this page

Reply:
Rather than hardcoding the style into the page, splitting the presentation (CSS) from the content (HTML), it's usually a better idea to introduce new styles that modify upon old styles, and place them alongside the other CSS.

Specifically, a `.custom-address .jupiter { ... }` entry in the CSS would allow you to place additional style information on the existing jupiter class, but only when it occurs as a descendant of a tag with a custom-address style. You can then set that style on some tag higher up the HTML for this page, and your custom rules will only apply within the confines of this page. It'd also let you avoid redefining the entirety of each style class, since the original `.jupiter` style will be applied first.

A better name than custom-address might be needed - it's hard to tell what's being modified here, precisely because the CSS is fragmented this way.